contracts/src/utils/cryptography/ecdsa.rs

```diff
@@ -8,7 +8,6 @@ use alloy_primitives::{address, uint, Address, B256, U256};
 use alloy_sol_types::{sol, SolType};
 use stylus_sdk::{
     call::{self, Call, MethodError},
-    storage::TopLevelStorage,
     stylus_proc::SolidityError,
 };
 
```

Comment:
Update _CHANGELOG.md_ with a short description of your changes in this PR.

Reply:
Updated `CHANGELOG.md`